### DISCUSSION TITLE
How to get Fields errors from useFieldArrays in remix-validation-form ?

### DISCUSSION THREAD

**Discussion by YassineOujaa77** [airjp73/rvf#293] [Q&A]
I'm trying to build a form using rremix-validation-form and I'm using zod for creating the schema validation
this is my schema :
```
z.object({
    subQuestions: z.array(
      z.object({
        id: z.string(),
        question: z.string().min(1),
        answers: z.array(
          z.object({
            id: z.string(),
            answer: z.string().min(1),
          })
        ),
      })
    ),
  })

```
but I cannot get the answers errors in my form ?
I searched in remix-validation-form and in useFieldArray there is no way I can get these fields' errors ?
DO you have any idea how can I do that ?

**Comment by airjp73** [comment]
Hi!

You would get the error using `useField` for the specific field like this:

```js
const { error } = useField("subQuestions[0].question");
```

**Reply by YassineOujaa77** [reply]
But how can I get the errors of the answers array ?

**Reply by airjp73** [reply]
Oh, you mean array-level errors? Looks like I forgot to document it, but you can grab that directly from `useFieldArray`.

```js
const [itemDefaults, helpers, error] = useFieldArray("subQuestions");
```
